app/src/main/java/ca/rmen/android/scrumchatter/chart/ChartUtils.java

```diff
@@ -54,14 +56,20 @@ static void setupYAxis(Context context, String yAxisLabel, Axis yAxis) {
 
     static void addLegendEntry(Context context, ViewGroup legendView, String name, int color) {
         TextView memberLegendEntry = new TextView(context);
-        memberLegendEntry.setTextColor(color);
         memberLegendEntry.setText(name);
         memberLegendEntry.setLayoutParams(new ViewGroup.LayoutParams(ViewGroup.LayoutParams.WRAP_CONTENT, ViewGroup.LayoutParams.WRAP_CONTENT));
         memberLegendEntry.setPadding(0, 0, context.getResources().getDimensionPixelSize(R.dimen.chart_legend_entry_padding), 0);
 
-        Drawable icon = ContextCompat.getDrawable(context, R.drawable.ic_legend_square);
+        Drawable icon = ContextCompat.getDrawable(context, R.drawable.ic_legend_square).mutate();
```

Comment:
Euuhh but unfortunately the "setTint" api isn't there on old devices.... so on old versions there is no color in the legend... facepalm